app/models/purchase_order/csv_builder.rb

```diff
@@ -1,36 +1,40 @@
 require 'csv'
 
-class PurchaseOrder::CsvBuilder
-  attr_reader :csv_lines
-  attr_reader :fields
-  attr_reader :purchase_orders
+# PurchaseOrder
+class PurchaseOrder
+  # Build CSV from purchase orders.
+  class CsvBuilder
+    attr_reader :csv_lines
+    attr_reader :fields
+    attr_reader :purchase_orders
 
-  def initialize
-    @csv_lines = []
-    @purchase_orders = []
-    @fields = PurchaseOrder::DenormalizedLine.
-      properties.
-      collect { |k, v| k }
-  end
-
-  def add(po)
-    if @csv_lines.empty?
-      @csv_lines << @fields.join(',')
+    def initialize
+      @csv_lines = []
+      @purchase_orders = []
+      @fields = PurchaseOrder::DenormalizedLine.
+        properties.
+        collect { |k, v| k }
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Prefer `map` over `collect`.